testdata/p4_16_bmv_errors/no-externs-bmv2.p4

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2016 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <v1model.p4>
+
+extern void extern_func(out bit<32> d, bit<32> s);
```

Comment:
I think you should also document this function explaining what the external implementation does.
No one can guess without reading all the code, which is scattered among several files.
You should also document here where the function implementation is found.

Reply:
Sure. Will do (here and in the positive test)